src/main/app/src/app/admin/zaakafhandel-parameters.service.ts

```diff
@@ -64,9 +64,11 @@ export class ZaakafhandelParametersService {
       );
   }
 
-  listResultaattypes(zaaktypeUuid: string): Observable<Resultaattype[]> {
+  listResultaattypes(zaaktypeUuid: string) {
```

Comment:
Why no explicit result type?

Reply:
we do this already by setting the type explicitly on `.get<GeneratedType<"RestResultaattype">[]>`, not need to add double the explicity type